src/components/AccessibilityBody/AccessibilityBody.scss

```diff
@@ -0,0 +1,3 @@
+.alphabetical-list {
+  list-style-type: lower-alpha;
+}
```

Comment:
Awesome! I didn't know this was a thing 👍

Reply:
Mark pointed it out to me!